classes/submission/genre/Repository.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * @file classes/submission/genre/Repository.php
+ *
+ * Copyright (c) 2014-2023 Simon Fraser University
+ * Copyright (c) 2000-2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Repository
+ *
+ * @brief A repository to find and manage genre.
+ */
+
+namespace PKP\submission\genre;
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+
+class Repository
+{
+    /**
+     * Retrieve a Genre by its ID (and optionally by context).
+     */
+    public function get(int $genreId, ?int $contextId = null): ?Genre
+    {
+        return Genre::findById($genreId, $contextId);
+    }
+
+    /**
+     * Retrieve a Genre by its entry_key (and optionally by context) using the model’s own finder.
+     */
+    public function getByKey(string $key, ?int $contextId = null): ?Genre
```

Comment:
Do we need this method and the method above in this helper? They are just calling the similar method of the Model